assets/app/features/upgrade/templates/upgrade7/nodes-repository-checks.less

```diff
@@ -0,0 +1,11 @@
+.upgrade-flow .upgrade-details-container .upgrade-details {
+	.nodes-repository-checks-page {
+		p {
+			margin-bottom: 20px;
+			display: inline-block;
+		}
+		.checks-container {
+			margin-top: 10px;
+		}
```

Comment:
I think we can replace those styles for the ones defined in `upgrade.less` (https://github.com/crowbar/crowbar-ui/pull/13/files#diff-b367b6ca6a6234ad236e1bab9fc55ea6) and `admin-repositories-checks-page.less` (https://github.com/crowbar/crowbar-ui/pull/13/files#diff-8bb0d3096440e53145feecbe610ee06c)